src/components/SuggestionsList/SuggestionsList.css

```diff
@@ -0,0 +1,23 @@
+/*! Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license. */
+@import "src/css/variables/colors.css";
+
+.y-suggestions-list {
+  width: 280px;
+}
+
+.y-suggestions-list--with-status .y-suggestions-list--loading {
+  height: 52px;
+}
+
+.y-suggestions-list--with-status.y-suggestions-list--with-results .y-suggestions-list--loading {
+  height: 47px;
```

Comment:
On the design, this is 48px including the line on the top of the row, hence 47.

Reply:
ok - ill regenerate the reference files